test/library/packages/LinearAlgebra/correctness/testSVD-error.good

```diff
@@ -0,0 +1,3 @@
+uncaught LinearAlgebraError: LinearAlgebra error : SVD computation did not converge. Returned non-zero value for "info":-6
```

Comment:
`Returned non-zero value for "info` -- "info" doesn't mean anything to me in this context and seems more like it's exposing an internal name?  If it's a common name and means something to LA folks, I'm fine with keeping it

Reply:
Yeah, that's a good point. I can translate the value to the meaning and try to give a better error message depending on the value.